pages.es/common/cargo.md

```diff
@@ -0,0 +1,37 @@
+# cargo
+
+> Gestiona proyectos de Rust y sus dependencias de módulos (crates).
+> Algunos subcomandos como `cargo build` tienen su propia documentación de uso.
+> Más información: <https://crates.io/>.
+
+- Buscar módulos:
+
+`cargo search {{texto_de_búsqueda}}`
+
+- Instalar un módulo:
```

Comment:
`crate` is a specific domain language that we define at the top level in the command description section. Why you don't use it here?
```suggestion
- Instala un crate:
```